avs_rvtools_analyzer/routes/api_routes.py

```diff
@@ -385,3 +392,114 @@ async def convert_excel_to_json(
 
         # Return simplified response with just the data
         return json_result
+
+    # Azure OpenAI Integration Endpoints
+
+    @app.post(
+        "/api/ai-suggestion",
+        tags=["AI Integration"],
+        summary="Get AI Risk Analysis Suggestion",
+        description="Get AI-powered suggestions for a specific migration risk using Azure OpenAI",
+        response_model=AISuggestionResponse,
+    )
+    async def get_ai_suggestion(request: AISuggestionRequest):
+        """Get AI-powered suggestions for a specific migration risk."""
+        try:
+            # Check if already configured via env vars, otherwise configure with request data
+            if not azure_openai_service.configured_via_env:
+                configured = azure_openai_service.configure(
+                    azure_endpoint=request.azure_endpoint,
+                    api_key=request.api_key
+                )
+                
+                if not configured:
+                    return AISuggestionResponse(
+                        success=False,
+                        error="Failed to configure Azure OpenAI client",
+                        risk_name=request.risk_name
+                    )
+            
+            # Get AI suggestion
+            suggestion = azure_openai_service.get_risk_analysis_suggestion(
+                risk_name=request.risk_name,
+                risk_description=request.risk_description,
+                risk_data=request.risk_data,
+                risk_level=request.risk_level,
+                deployment_name=request.deployment_name
+            )
+            
+            if suggestion:
+                return AISuggestionResponse(
+                    success=True,
+                    suggestion=suggestion,
+                    risk_name=request.risk_name
+                )
+            else:
+                return AISuggestionResponse(
+                    success=False,
+                    error="Failed to generate AI suggestion",
+                    risk_name=request.risk_name
+                )
+                
+        except Exception as e:
+            return AISuggestionResponse(
+                success=False,
+                error=f"Error generating AI suggestion: {str(e)}",
+                risk_name=request.risk_name
+            )
+
+    @app.post(
```

Comment:
We will remove the possibility for the user to configure AI integration from the UI. Only keep the method with environment variables and server side configuration.